### ISSUE TITLE
DeprecationWarning in eslint 5

### ISSUE THREAD

**Issue by henri-hulski** [feross/eslint-config-standard-jsx#32]
When upgrading to eslint 5 I get the following DeprecationWarnings:
```
(node:7485) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread
' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard-jsx")
(node:7485) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread
' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```
So the same for `eslint-config-standard`.

**Comment by henri-hulski** [comment]
See https://github.com/standard/eslint-config-standard/issues/123#issuecomment-401525282

**Comment by feross** [comment]
This is fixed in `eslint-config-standard-jsx` 6.0.0!
